packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -1,11 +1,13 @@
 import Nat from '@agoric/nat';
 import { assert, details } from '@agoric/assert';
+import { makeNotifierKit } from '@agoric/notifier';
 
 export function buildRootObject(vatPowers) {
   const { D } = vatPowers;
   const repeaters = new Map();
 
   async function createTimerService(timerNode) {
+    /* @type {TimerService} */
     return harden({
```

Comment:
The types don't seem to work for me if we harden an object immediately. I think it is better to create the typed object separately, then harden and return the object. Can you confirm that you get the method types for TimerService coming through in VSCode?

```suggestion
    const timerService = {
```

Reply:
fixed.  typescript and vscode seem happy.